frontend/src/components/mine/MineDashboard.js

```diff
@@ -148,6 +155,13 @@ export class MineDashboard extends Component {
                     />
                   </div>
                 </TabPane>
+                {mine.major_mine_ind && (
+                  <TabPane tab="Applications" key="applications">
+                    <div className="tab__content">
+                      <MineApplicationInfo mine={mine} {...this.props} />
```

Comment:
As per a brief thread with @nayr974 , we'd like to be more explicit with the props that are being passed in and move away from the blind passing of all `props`

Reply:
Agree here. Let's avoid new uses of {...this.props} and start refactoring existing uses.

Reply:
Done.